lib/nifty_services/configuration.rb

```diff
@@ -1,25 +1,23 @@
-require 'logger'
-
 module NiftyServices
   class Configuration
 
-    DEFAULT_I18N_NAMESPACE = "nifty_services"
-    DEFAULT_SERVICE_CONCERN_NAMESPACE = 'NitfyServices::Concerns'
+    DEFAULT_I18N_NAMESPACE = 'nifty_services'.freeze
+    DEFAULT_SERVICE_CONCERN_NAMESPACE = 'NitfyServices::Concerns'.freeze
 
     ERROR_RESPONSE_STATUS = {
-      :bad_request           => 400,
-      :not_authorized        => 401,
-      :forbidden             => 403,
-      :not_found             => 404,
-      :unprocessable_entity  => 422,
-      :internal_server       => 500,
-      :not_implemented       => 501
-    }
+      bad_request:          400,
+      not_authorized:       401,
+      forbidden:            403,
+      not_found:            404,
+      unprocessable_entity: 422,
+      internal_server:      500,
+      not_implemented:      501
+    }.freeze
```

Comment:
@viniciuszago This will throw a `RuntimeError: can't modify frozen Hash` when adding a new error method. This will break codebase, but I'm not happy with `add_error_method` modifying a constant value too, so this(hash freezing) can be done after this issue is resolved.